salt/elasticsearch/init.sls

```diff
@@ -5,12 +5,11 @@ elasticsearch-deps:
 elasticsearch:
     # add repo to managed repositories
     pkgrepo.managed:
-        - name: deb https://artifacts.elastic.co/packages/7.x/apt stable main
+        - name: deb https://artifacts.elastic.co/packages/7.6.1/apt stable main
```

Comment:
I couldn't find a repo for this, to pin the version I think we need to specify the version to `pkg.installed` instead. Docs here: https://docs.saltstack.com/en/latest/ref/states/all/salt.states.pkg.html#salt.states.pkg.installed